lessonbuilder/tool/src/webapp/templates/ShowPage.html

```diff
@@ -530,7 +530,7 @@ <h5 class="modal-title" id="subpageLabel" rsf:id="msg=simplepage.subpage">simple
             </div>
             <div class="modal-footer">
               <button class="btn-primary" type="submit" rsf:id="create-subpage" onclick="event.returnValue = checkSubpageForm(); return checkSubpageForm();"></button>
-              <button class="btn btn-secondary" type="button" id="subpage-choose" rsf:id="subpage-choose" style="display:none;"><span rsf:id="msg=simplepage.choose_existing_page"></span></button>
+              <a class="btn btn-secondary" id="subpage-choose" rsf:id="subpage-choose" style="display:none;"><span rsf:id="msg=simplepage.choose_existing_page"></span></a>
```

Comment:
should the link have the attribute role="button"?

Reply:
Probably, for the a11y. Well spotted, chief :)